config/core/resources/broker.yaml

```diff
@@ -54,6 +54,93 @@ spec:
     name: v1
     served: true
     storage: false
+    schema:
+      openAPIV3Schema:
+        type: object
+        description: "Broker collects a pool of events that are consumable using Triggers. Brokers provide a well-known endpoint for event delivery that senders can use with minimal knowledge of the event routing strategy. Receivers use Triggers to request delivery of events from a Broker's pool to a specific URL or Addressable endpoint."
+        properties:
+          spec:
+             description: "Spec defines the desired state of the Broker."
+             type: object
+             properties:
+              config:
+                description: "Config is a KReference to the configuration that specifies
+                      configuration options for this Broker. For example, this could
+                      be a pointer to a ConfigMap."
+                type: object
+                properties:
+                  apiVersion:
+                    description: "API version of the referent."
+                    type: string
+                  kind:
+                    description: "Kind of the referent. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds"
+                    type: string
+                  name:
+                    description: "Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names"
+                    type: string
+                  namespace:
+                    description: "Namespace of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/
+                        This is optional field, it gets defaulted to the object
+                        holding it if left out."
+                    type: string
+              delivery:
+                description: "Delivery is the delivery specification for Events within
+                    the Broker mesh. This includes things like retries, DLQ, etc."
+                type: object
+                properties:
+                  backoffDelay:
+                    description: 'BackoffDelay is the delay before retrying. More
```

Comment:
nit: is there any convention in using single or double quotes in description?

Reply:
decided to use single quotes as its can be applied homogeneously (i.e., double quotes are not good in some cases)